app/test_main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+from unittest import mock
```

Comment:
Better use 1 space between this two types of imports, but in general code looks good